nnsvs/gen.py

```diff
@@ -198,27 +250,48 @@ def predict_acoustic(device, labels, acoustic_model, acoustic_in_scaler,
     # Predict acoustic features
     x = torch.from_numpy(linguistic_features).float().to(device)
     x = x.view(1, -1, x.size(-1))
-    pred_acoustic = acoustic_model(x, [x.shape[1]]).squeeze(0).cpu().data.numpy()
 
-    # Apply denormalization
-    pred_acoustic = acoustic_out_scaler.inverse_transform(pred_acoustic)
+    if acoustic_model.prediction_type == "probabilistic":
```

Comment:
Same as above.

Reply:
I changed to use  predictin_type() of BaseModel in the latest commit.